src/carbons.c

```diff
@@ -179,7 +180,10 @@ void carbons_xml_stripped_cb(PurpleConnection * gc_p, xmlnode ** stanza_pp) {
   }
 
   purple_debug_info(CARBONS_LOG_CATEGORY, "Writing body of the carbon copy of a sent message to the conversation window with %s.\n", buddy_name_bare);
-  purple_conversation_write(conv_p, xmlnode_get_attrib(*stanza_pp, "from"), xmlnode_get_data(body_node_p), PURPLE_MESSAGE_SEND, time((void *) 0));
+  body_data = xmlnode_get_data(body_node_p);
+  purple_conversation_write(conv_p, xmlnode_get_attrib(*stanza_pp, "from"), body_data, PURPLE_MESSAGE_SEND, time((void *) 0));
+  g_free(body_data);
```

Comment:
nitpicking, but could you put this `g_free()` with the other `free`s by moving it one line down or something :slightly_smiling_face:

Reply:
same as above